dashboard/src/components/Catalog/Catalog.test.tsx

```diff
@@ -310,6 +310,18 @@ describe("filters by the searched item", () => {
 });
 
 describe("filters by application type", () => {
+  let spyOnUseDispatch: jest.SpyInstance;
+  const mockDispatch = jest.fn();
+
+  beforeEach(() => {
+    spyOnUseDispatch = jest.spyOn(ReactRedux, "useDispatch").mockReturnValue(mockDispatch);
+  });
+
+  afterEach(() => {
+    spyOnUseDispatch.mockRestore();
+    mockDispatch.mockReset();
+  });
+
```

Comment:
This was common for a few components which you'd converted to functions: they no longer had a `dispatch` prop, so we have to spy on the one used in the new function (via `useDispatch`).